cedar-policy-core/src/parser/cst.rs

```diff
@@ -60,6 +60,16 @@ pub struct Policy {
     pub conds: Vec<Node<Cond>>,
 }
 
+#[derive(Debug, Clone, PartialEq, Eq)]
+/// Policy statement - can be an Error node when 'tolerant-ast' feature is switched on
+pub enum Policy {
+    /// Policy that has been parsed successfully
```

Comment:
Nit, `Policy` can be created through other means than parsing

Reply:
Good point, will fix